### ISSUE TITLE
load checkpoint from local path: checkpoints/tsn**42.pth 段错误 (核心已转储)

### ISSUE THREAD

**Issue by a14147397** [open-mmlab/mmaction2#2100]
I have searched related issues but cannot get the expected help.

我的错误是：
My error is:
load checkpoint from local path: checkpoints/tsn**42.pth 段错误 (核心已转储)

**Describe the bug**
I 'm tring to run demo.py with the following command.
directly copy from [README.md](https://github.com/open-mmlab/mmaction2/tree/master/demo#readme) Example

`# The demo.mp4 and label_map_k400.txt are both from Kinetics-400
python demo/demo.py configs/recognition/tsn/tsn_r50_video_inference_1x1x3_100e_kinetics400_rgb.py \
    checkpoints/tsn_r50_1x1x3_100e_kinetics400_rgb_20200614-e508be42.pth \
    demo/demo.mp4 tools/data/kinetics/label_map_k400.txt --out-filename demo/demo_out.mp4`
![2022-11-30_10-07](https://user-images.githubusercontent.com/80265033/204698023-b91268a9-9200-4a4e-919c-3640f9b1dde8.png)

我使用官网提供的模型
My model downloaded with officail net
https://download.openmmlab.com/mmaction/recognition/tsn/tsn_r50_1x1x3_100e_kinetics400_rgb/tsn_r50_1x1x3_100e_kinetics400_rgb_20200614-e508be42.pth

**Environment**
我的环境是根据readme中的quickstart指导用conda创建的Python=3.8，pytorch版本为与cuda11.7匹配，更新为1.13.版本，使用pip命令安装，torch.cuda.is_available()为True, mmcv-full =1.7.0
My environment is installed according to quickstart,python =3.8, pytorch=1.13，cuda=11.7，use with pip,torch.cuda.is_available() is true, mmcv-full =1.7.0

补充：
我的系统是ubuntu22.04，当使用pycharm配置参数运行上述命令时，对应的报错信息如下：
Supplement: 
My system is ubuntu 22.04. When the above command is run with pycharm configuration, the corresponding error message is as follows:
`load checkpoint from local path: ../checkpoints/tsn_r50_1x1x3_100e_kinetics400_rgb_20200614-e508be42.pth

Process finished with exit code 139 (interrupted by signal 11: SIGSEGV)
`
![image](https://user-images.githubusercontent.com/80265033/204731796-cd96bbfb-68d7-489c-90d4-0f251725a8bc.png)

可能是什么原因啊，我的模型文件位置没错。
What could be the reason? my checkpoint file stay in the correct path.

**Error traceback**
```
A placeholder for traceback.
```

**Comment by cir7** [comment]
Hi, @a14147397 . I have no idea what caused the bug, you can try re-install mmcv with 
`pip install mmcv-full==1.7.0 -f https://download.openmmlab.com/mmcv/dist/cu117/torch1.13/index.html`

**Comment by Mactarvish** [comment]
same error

**Comment by cir7** [comment]
Hi, @Mactarvish , have you tried re-install mmcv? or can you set some breakpoint to figure out which line the code is crash at

**Comment by Mactarvish** [comment]
> Hi, @Mactarvish , have you tried re-install mmcv? or can you set some breakpoint to figure out which line the code is crash at

I've tried mmcv 1.5 and 1.7 on `769004c` and both of them will crash, which is caused by `inference.py` : line 51: `model.to(device)`.

**Comment by cir7** [comment]
@Mactarvish , try comment [line 7](https://github.com/open-mmlab/mmaction2/blob/master/demo/demo.py#L7) in demo.py

**Comment by cir7** [comment]
@Mactarvish @a14147397 Thanks for your feedback, the bug seems caused by the incompatible of `deocrd`, we already fix it in pr #2104 , we will merge it asap.

**Comment by a14147397** [comment]
@cir7 Thanks, I fix demo bug with your pr #2104 , but new bug arised. I use mmcv-full==1.7.0, could you recommend the version to me? My cuda=11.7, torch=1.13, this environment should be feasible. I debug out the error:
RuntimeError: The type of argument video is not supported: <class 'str'>

![2022-12-02_21-07](https://user-images.githubusercontent.com/80265033/205304995-fd3a98f9-e686-4306-be92-8aefcc62fcb8.png)
![2022-12-02_21-09](https://user-images.githubusercontent.com/80265033/205305009-c9b42e05-ec13-4a6d-82d3-68489d4b4106.png)

**Comment by cir7** [comment]
@Mactarvish it seems that you are located at wrong directory and cause check `osp.exists(video)` fail, please make sure that you are located at `$MMACTION2` and try run following commands:
   ```shell
   # The demo.mp4 and label_map_k400.txt are both from Kinetics-400
   python demo/demo.py configs/recognition/tsn/tsn_r50_video_inference_1x1x3_100e_kinetics400_rgb.py \
       checkpoints/tsn_r50_1x1x3_100e_kinetics400_rgb_20200614-e508be42.pth \
       demo/demo.mp4 tools/data/kinetics/label_map_k400.txt
   ```

**Comment by a14147397** [comment]
Everybody, I solved this question, I reinstalled Ubuntu and CUDA. I conferred this question could be a cuda incompatibile problem. The fact is that I run out this demo with ubuntu18.04 & cuda11.4. I guess it's because ubuntu22.04's kernel version too high for cuda 11.7&11.X  to support.   I almost reinstalled all 11.x cuda in ubuntu22.04. 

![图片](https://user-images.githubusercontent.com/80265033/205652178-a8686acd-402b-4346-baec-004b200cf0ee.png)
